### ISSUE TITLE
The performance on PASCAL VOC val set

### ISSUE THREAD

**Issue by zhijiejia** [Visual-Attention-Network/SegNeXt#12]
Excuse me, can you tell me the performance results of segnext on the validation set of pascal voc? I found that both PSPNet and Deeplabv3 + have 85+ mIoU on the test set, but only 80 on the validation set (according to the mmsegmentation repository), so I want to confirm whether the result on the test set will be higher than that on the validation set ？

**Comment by MenghaoGuo** [comment]
Yes, you are right.

The performance on test set is higher than that in val set. 

I think the reason for this may be that the data in the validation set is additionally used in the training process before testing the test set

**Comment by zhijiejia** [comment]
ok，fine !  Can you tell segnext's performence on the validation set of pascal voc, which the performance on the test set reach to 90.6

**Comment by zhijiejia** [comment]
Hi, I would like to ask, is the voc data set in the paper in segnext a voc2012 data set? Why do I download the voc2012test data set with 16135 images, far more than 1456 images, I downloaded from here (http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2012test.tar), may I ask where did you download the testData set from?

**Comment by zhijiejia** [comment]
ok, I find the test.txt only have 1456 records ... hahaha

**Comment by MenghaoGuo** [comment]
hahaha

**Comment by longchanyuan** [comment]
> Excuse me, can you tell me the performance results of segnext on the validation set of pascal voc? I found that both PSPNet and Deeplabv3 + have 85+ mIoU on the test set, but only 80 on the validation set (according to the mmsegmentation repository), so I want to confirm whether the result on the test set will be higher than that on the validation set ？

How did you perform on pascal voc in segnext, what pre-training weights were used, I used MSCAN-B, but there were only a dozen mious out of the run, I don't know what went wrong
